config/setup.py

```diff
@@ -1,7 +1,7 @@
 import setuptools
 from setuptools.dist import Distribution
 
-_VERSION = "0.1.6"
+_VERSION = "0.7.0"
```

Comment:
add modifying the version here in tools/RELEASE.md, step 2?

Reply:
*Done